<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="welcome_label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Welcome to Atef!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="left_vlayout">
       <item>
        <widget class="QLabel" name="start_label">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Start:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="new_passive_button">
         <property name="text">
          <string>New Passive Checkout</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="new_active_button">
         <property name="text">
          <string>New Active Checkout</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="open_button">
         <property name="text">
          <string>Open ...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="left_vspacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exit_button">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="right_vlayout">
       <item>
        <widget class="QLabel" name="examples_label">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Examples</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sample_passive_button">
         <property name="text">
          <string>Sample Passive Checkout</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sample_active_button">
         <property name="text">
          <string>Sample Active Checkout</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="docs_label">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Documentation:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="docs_button">
         <property name="text">
          <string>Open Documentation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="source_button">
         <property name="text">
          <string>View source</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="right_vspacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
